images/win/scripts/Installers/Install-Toolset.ps1

```diff
@@ -48,7 +48,7 @@ $ErrorActionPreference = "Stop"
 Import-Module -Name ImageHelpers -Force
 
 # Get toolcache content from toolset
-$tools = Get-ToolsetContent | Select-Object -ExpandProperty toolcache
+$tools = Get-ToolsetContent | Select-Object -ExpandProperty toolcache | Where-Object url

```

Comment:
Honestly, I don't like this condition and current implementation because Install-Toolset starts to fail when something is added to toolset.json.
I suggest to define the list of tools here `$ToolsToInstall = @("Python", "Node")` and get only them.